src/pages/index.tsx

```diff
@@ -7,76 +7,55 @@ import { resultType } from "../utils/resultType";
 import React from "react";
 
 const Home: React.FC<{ results: resultType[] }> = ({ results }) => {
+
+  results.sort((a, b) => b.score - a.score);
+
   return (
-    <div className={styles.container}>
+    <main className={styles.container}>
       <Head>
         <title>MICP Leaderboard | TPC</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />  
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <Heading />
-      <table className={styles.table}>
-        <tbody>
-          <TableHeadings />
-          {results.map((e, i) => (
-            <TableRow key={i} {...e} />
-          ))}
-        </tbody>
-      </table>
-    </div>
+      <section className={styles.rankSection}>
+        {results.map((e, i) => (
+          <TableRow
+            key={i}
+            rank={i+1}
+            username={e.username}
+            name={e.name}
+            score={e.score}
+            currentRating={e.currentRating}
+          />
+        ))}
+      </section>
+    </main>
   );
 };
 
-export async function getStaticProps() {
-  const body = {
-    query:
-      "query {allMembers(_size: 1000) {data {username, initialRating, currentRating}}}",
-  };
-  const res = await fetch(process.env.DB_URL!, {
-    method: "POST",
-    headers: {
-      Authorization: "bearer " + process.env.DB_SECRET,
-    },
-    body: JSON.stringify(body),
-  });
-  const rawData = await res.json();
-  if (!rawData) {
+export async function getServerSideProps() {
+  const response = await fetch('https://micp-backend.onrender.com/api/rankList', {
```

Comment:
We should store this as an environment variable otherwise anyone can spam this endpoint.